packages/gast/src/model.ts

```diff
@@ -263,10 +230,7 @@ export class Terminal implements IProductionWithOccurrence {
     label?: string
     idx?: number
   }) {
-    assign(
-      this,
-      pickBy(options, (v) => v !== undefined)
-    )
```

Comment:
This function, as well as the ones in the chunks above, seem to work fine without `pickBy`. It would've been noisy to insert an inline version of `pickBy` several times, so opted to go with `Object.assign(this, options)`, thus assigning fields even if they're `undefined` in `options`